src/main/java/uk/gov/companieshouse/api/accounts/AccountsApplication.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse.api.accounts;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+@SpringBootApplication
+public class AccountsApplication extends WebMvcConfigurerAdapter {
+
+    public static void main(String[] args) {
+
+        // Returns the configured ${PORT} value in the `start.sh` script - passed in under `server.port`.
+        // If no port is configured, return null
+        Integer port = Integer.getInteger("server.port");
+
+        if (port == null) {
+            // When logging has been added, this print line will be replace
+            System.out.println("Failed to start service, no port has been configured");
+            System.exit(0);
```

Comment:
This should be logged using structured logging.

Reply:
done